src/time/time.zig

```diff
@@ -556,6 +556,26 @@ pub const Duration = struct {
     }
 };
 
+pub const Instant = struct {
+    inner: std.time.Instant,
+
+    pub fn now() Instant {
+        return .{ .inner = std.time.Instant.now() catch @panic("std.time.Instant unsupported!") };
```

Comment:
Why not return an error here? Are we sure that every platform we want to target is going to support this? In that case, it would probably be faster to use unreachable here (not sure how much that matters).

I think it would be good to have a standard comment anywhere we have an explicit panic or UB for unsupported platforms. Something like `// this is unreachable on supported platforms`? Maybe the docs should also specify the supported platforms.